htmd/queues/localqueue.py

```diff
@@ -117,7 +154,9 @@ def submit(self, mydirs):
         """
         self._setupQueue()
 
-        if isinstance(mydirs, str): mydirs = [mydirs]
+        if isinstance(mydirs, str):
+            mydirs = [mydirs]
+        self._dirs.extend(mydirs)
```

Comment:
why here and not in .submit?

Reply:
oh right, this is a bug, I'll change it.

Reply:
ah forget it. bad diff viewer